aws/v12.1.0/README.md

```diff
@@ -0,0 +1,23 @@
+# :zap: Giant Swarm Release v12.1.0 for AWS :zap:
+
+**If you are upgrading from 12.0.0, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+## Change details
+
+#### Significant changes
+
+Cert Manager is upgraded from upstream version v0.9.0 to v0.15.2. Due to changes in Cert Manager's API,
+associated Ingresses and Secrets must also be updated to ensure they are reconciled by Cert Manager. An
+[upgrade script](https://github.com/giantswarm/cert-manager-app/blob/master/files/migrate-v090-to-v200.sh)
+is provided to assist with the process.
+
+**This is a breaking change.** Please ensure you have read the [upgrade notes](https://github.com/giantswarm/cert-manager-app#upgrading-from-v090-giant-swarm-app-v108-to-20x).
+
+**Note for SEs:** when upgrading clusters to this release, ensure the App is also cordoned on the installation's
```

Comment:
@glitchcrab This notes applies to anyone upgrading from v1.0.8 to any v2.0.x, right? If that is the case, this note is better in the release notes and not here?

Reply:
yup, removed it.